pypeit/core/coadd.py

```diff
@@ -2087,7 +2087,7 @@ def combspec(waves, fluxes, ivars, masks, sn_smooth_npix,
     ivars = np.float64(ivars)
 
     # Generate a giant wave_grid
-    wave_grid, _, _ = wvutils.get_wave_grid(waves, masks = masks, wave_method=wave_method,
+    wave_grid, wave_grid_mid, _ = wvutils.get_wave_grid(waves, masks = masks, wave_method=wave_method,
```

Comment:
You've changed the return values but did not document the new quantity. Please update the docs evrerywhere where you made such changes.

Reply:
Done.